src/common/components/nav/navCart/navCart.component.js

```diff
@@ -43,7 +43,9 @@ class NavCartController{
           this.loading = false;
           this.hasItems = !isEmpty(this.cartData.items);
           this.analyticsFactory.buildProductVar(data);
-          this.analyticsFactory.setEvent('cart open');
+          if(this.cartData.items.length === 1){
+            this.analyticsFactory.setEvent('cart open');
```

Comment:
Yes, that's the intent.  I checked there, but it will require an extra request for non public users.  I don't want to add that.  I'll check with Brenton.